src/biz/bokhorst/xprivacy/XAppWidgetManager.java

```diff
@@ -42,12 +42,15 @@ public String getClassName() {
 		getInstalledProviders, getInstalledProvidersForProfile, Srv_getInstalledProviders, Srv_getInstalledProvidersForProfile
 	};
 
-	public static List<XHook> getInstances() {
+	public static List<XHook> getInstances(boolean services) {
```

Comment:
Yes, I did not see that you had a few hooks with a similar argument to begin with.